src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,53 @@
+package racingcar.domain;
+
+public class Car {
+
+    private static final int MIN_LENGTH = 1;
+    private static final int MAX_LENGTH = 5;
+    private static final int CAN_MOVE_CONDITION = 4;
+    private static final String NAME_LENGTH_EXCEPTION = "자동차 이름은 %d ~ %d글자만 가능합니다.";
+    private static final String NAME_LETTER_EXCEPTION = "자동차 이름에 특수문자나 숫자는 포함될 수 없습니다.";
+
+    private final String name;
+    private int moveCount;
+
+    public Car(String name) {
+        name = name.trim();
+        validateLength(name);
+        validateLetter(name);
+        this.name = name;
+        this.moveCount = 0;
+    }
+
+    private void validateLength(String name) {
+        if (name.length() < MIN_LENGTH || MAX_LENGTH < name.length()) {
+            throw new IllegalArgumentException(String.format(NAME_LENGTH_EXCEPTION, MIN_LENGTH, MAX_LENGTH));
+        }
+    }
+
+    private void validateLetter(String name) {
+        for (char c : name.toCharArray()) {
+            if (!Character.isLetter(c)) {
+                throw new IllegalArgumentException(NAME_LETTER_EXCEPTION);
+            }
```

Comment:
이 부분도 아래와 같이 스트림을 이용할 수 있을 것 같은데 반복문을 사용하신 이유가 있을까요?
```suggestion
if(!name.chars().allMatch(Character::isLetter) {
    throw new IllegalArgumentException(NAME_LETTER_EXCEPTION);
}
```

Reply:
`allMatch()`를 활용할 수 있군요..!!!
해당 api는 사용해본 적이 없어서 몰랐어요! 이제부터 유용하게 써먹어 보겠습니다!!!👍👍